src/App.tsx

```diff
@@ -1,11 +1,90 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import * as todoServices from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Filter } from './types/Filter';
+import classNames from 'classnames';
+import { client } from './utils/fetchClient';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [loading, setLoading] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [newTodoTitle, setNewTodoTitle] = useState<string>('');
```

Comment:
If you use `string`, `number`, `boolean`, `undefined` and `null` you don't need to set a type